README.md

```diff
@@ -43,562 +49,131 @@ There are two interactive tutorials provided,
 A Lush theme template is available in the examples folder, as well as other
 examples for various topics (Lightline, dependency injection, etc).
 
-Table of Contents
------------------
-
-Lush broadly has 3 components,
-
-- [A HSL color manipulator](#hsl-colors)
-- The [lush-spec parser and compiler](#lush-spec)
-  - The [lush-spec spec](#lush-spec-spec)
-  - [Additional information](#additional-information) about lush-specs
-- [Lush.ify](#lushify), a buffer highlighting and hot-reload tool
-
-See also:
-
-- [Todo / Future Ideas](#todo--future-ideas)
-- [Change Log](#change-log)
-
-For a usage example, see the quick start, tutorial and examples folder.
-
-HSL Colors
-----------
 
-The [HSL color](https://www.w3.org/wiki/CSS3/Color/HSL) manipulator can be
-accessed via `require('lush').hsl`.  It may also be included in other modules
-via `require('lush.hsl')`.
+Usage
+-----
 
-You can create HSL colors by providing hue, saturation and lightness values,
-or providing a hexadecimal string.
+Creating a color scheme in Lush is a three step process:
 
-```lua
-color = hsl(0, 100, 50) -- equivilent to rgb(255,0,0) elsewhere
-hex_color = hsl("#FF0000") -- hex_color == color
-```
-
-It provides multiple methods for manipulating a color. All functions are pure,
-always returning new colors and leaving the originals unmodified. Functions
-can be chained.
-
-There are 3 main operations you may want to perform:
-
-- Rotate a hue.
-- Saturate (or desaturate) a color.
-- Lighten (or darken) a color.
+1. Define your colors using the HSL module
+2. Define your highlight groups with a Lush spec
+3. Organise your color scheme for normal usage and/or distribution
```

Comment:
@savq

I understand the order here, not getting stuck in file/folder creation before getting to play around, but I think not getting the correct setup at the start leads to bad inertia later on.

There is the lush-template folder in the examples folder, maybe step zero might be copying that folder somewhere and doing some basic renames? A problem with this is that people need to find the folder which may be unclear when you're using a plugin manager.

I did consider, and maybe regret not doing it now,  creating the template in its own repo, which would give "lush theme index" "for free" via the gh forks list. Maybe I will revisit that idea.